newdle/newdle.cfg.example

```diff
@@ -97,3 +97,6 @@ LAST_ACTIVITY_CLEANUP_DELAY = None
 FINAL_DATE_CLEANUP_DELAY = None
 # Days after which a newdle that was deleted by its creator is cleaned up.
 DELETED_CLEANUP_DELAY = None
+
+# The INDICO_URL is used for creating an event from a Newdle. 
```

Comment:
IIRC we said we wanted to make this generic at some point, ie `CREATE_EVENT_URL` with no mention of Indico or just as an example.

> If you use Indico, the URL to use is https://yourindico/event/prepare

Or something like that.

Reply:
Alright, updated to `CREATE_EVENT_URL`.